data/registry/application-integration-java-keycloak.yml

```
@@ -0,0 +1,21 @@
+# cSpell:ignore keycloak
+title: Keycloak
+registryType: application integration
+language: java
+tags:
+  - keycloak
+  - iam
+urls:
+  website: https://www.keycloak.org
+  repo: https://github.com/keycloak/keycloak
+  docs: https://www.keycloak.org/server/tracing
+license: Apache 2.0
+description:
+  Keycloak utilizes OpenTelemetry for distributed tracing provided by the
+  Quarkus OpenTelemetry extension.
+authors:
+  - name: Keycloak Authors
+    url: https://github.com/keycloak
```

Perhaps something like this would be more specific?
```suggestion
  - name: Keycloak Contributors
    url: https://github.com/keycloak/keycloak/graphs/contributors
```

---

At the end, the URL must be a valid user profile URL or organization URL.